server/cmd/gram/deps.go

```diff
@@ -58,6 +59,23 @@ type dbClientOptions struct {
 	enableUnsafeLogging bool
 }
 
+func newClickhouseClient(connstring string) (clickhouse.Conn, error) {
+	conn, err := clickhouse.Open(&clickhouse.Options{
+		Auth: clickhouse.Auth{
+			Database: "gram_metrics",
```

Comment:
These should probably not be hardcoded. Or if this is intended to be the connection path only for local development we should take environment in here.

Reply:
True